coral/src/app/features/configuration/clusters/components/ClustersTable.tsx

```diff
@@ -87,15 +103,15 @@ const ClustersTable = (props: ClustersTableProps) => {
     },
   ];
 
-  if (handleShowModal !== undefined) {
+  if (isAdminUser) {
```

Comment:
I would add a test case for that, too, so we make sure the "Remove" functionality is only available for admins.

Reply:
Sorry for not being more precise -> I meant explicitly confirming that there is no delete action available for the user without the permission (e.g. in the block `shows all clusters as a table (user without permissions.addDeleteEditClusters)`). It's can get a bit trick to decide how much of "testing that something is NOT there" makes sense 😅  but in a case of this kind of functionality I would do it.

Reply:
Aaah of course!

Added here ^^ [62dd1c7](https://github.com/Aiven-Open/klaw/pull/2354/commits/62dd1c72c5e18954b2b4958c01e58575c39218e9)